concordium-std/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 ## Unreleased changes
 
+- Add a new primitive `get_random`. It is used to generate random numbers in on-chain code for testing with QuickCheck.
```

Comment:
Perhaps we should use `Wasm` instead of on-chain code?

Reply:
@Bargsteen my intention was to emphasize that this is "special" code. As far as I understand, Wasm supports random number generation in general. But maybe it's enough context to understand it by just writing "Wasm".

Reply:
What do you mean that "Wasm" supports random number generation?

Wasm itself does not. It has to be made available via host functions. THere are no instructions to do this in Wasm.

Reply:
My bad, let's use "Wasm" then

Reply:
Fixed now.